packages/cli/commands/upload.js

```diff
@@ -55,7 +55,7 @@ exports.handler = async options => {
   await loadAndValidateOptions(options);
 
   if (!validateMode(options)) {
-    process.exit(EXIT_CODES.ERROR);
+    process.exit(EXIT_CODES.WARNING);
```

Comment:
Seeing `WARNING` as an exit code value feels kind of off since these really are errors — they just aren't internal. We also should be mindful of exit codes that `yargs` produces so that there is consistency, especially in its own validation of user input.

Reply:
Based off of the current description for exit code, we have `EXIT_CODES.ERROR` defined as
"Config problem or internal error". Should we treat invalid user input as a config issue (i.e. keep this as an error)?

Warnings are defined as `Warnings or validation issues`, which I'm realizing is pretty vague. I suppose this could also fall into that category too.

Reply:
Looked at #570 and I _think_ this is correct, but I am not really sure. The HubSpot CLI is more broad in terms of purpose than `eslint` so maybe we need to be clearer on when different codes should be used. I could also try and add more nuance into capturing why a given upload failed so that we're treating 400s due to validation errors differently than other failure states.